### PR TITLE
Add --variants-only flag.

### LINKED ISSUE #7: Enable -variantsonly mode
It would be great if we could specify that a VCF file be ingested into a dataset that only has the variant information, and none of the actual genotype calls.

Also, what happens when loading a VCF that contains no genotypes?  (I'm guessing it will deal with it correctly.)

### PR DESCRIPTION
**Pull request by tomwhite**
Fixes #7.

### CODE REVIEW

File: scripts/download_dbsnp.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the
+# License.
+
+hadoop fs -mkdir dbsnp
+
+hadoop fs -rm dbsnp/00-All.vcf
+curl ftp://ftp.ncbi.nih.gov/snp/organisms/human_9606/VCF/00-All.vcf.gz \
+  | gunzip -c \
+  | hadoop fs -put - dbsnp/00-All.vcf
```
Comment (laserson) [inline scripts/download_dbsnp.sh]: nit: newline?

File: src/main/java/com/cloudera/science/quince/FlattenVariantFn.java [hunk 2]
```diff
@@ -48,23 +58,29 @@ public static FlatVariantCall flatten(Variant variant, Call call) {
...
-    flatVariantCall.setGenotype1(get(call.getGenotype(), 0));
-    flatVariantCall.setGenotype2(get(call.getGenotype(), 1));
-    // call.getPhaseset(); TODO: ignored for now
-    flatVariantCall.setGenotypeLikelihood1(get(call.getGenotypeLikelihood(), 0));
-    flatVariantCall.setGenotypeLikelihood2(get(call.getGenotypeLikelihood(), 1));
-    //call.getInfo(); TODO: ignored for now
+    if (call != null) {
+      flatVariantCall.setCallSetId(call.getCallSetId());
+      flatVariantCall.setCallSetName(call.getCallSetName());
+      flatVariantCall.setVariantId(call.getVariantId());
```
Comment (laserson) [inline src/main/java/com/cloudera/science/quince/FlattenVariantFn.java]: What is this one again?  Should it be in the variant-only case?

**Comment by laserson** [comment]
Generally lgtm.

**Comment by laserson** [comment]
Merge away

**Comment by laserson** [comment]
The only thing I'll add is that the names of the classes are starting to get a little wonky and difficult to understand what's going on.

File: scripts/download_dbsnp.sh (hunk 1 above)
Comment (tomwhite) [inline scripts/download_dbsnp.sh]: Fixed

File: src/main/java/com/cloudera/science/quince/FlattenVariantFn.java (hunk 2 above)
Comment (tomwhite) [inline src/main/java/com/cloudera/science/quince/FlattenVariantFn.java]: No, this is inlined from call, so not needed in the variant-only case (use `variant.getId()` to get the variant ID).

**Comment by tomwhite** [comment]
Opened https://github.com/cloudera/quince/issues/28 to discuss naming improvements.

**Pull request merged**